package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@kiali/kiali-ui",
```

Comment:
We need `@kiali/kiali-ui` to be able to publish it to [npm kiali's account](https://www.npmjs.com/package/@kiali/kiali-ui)

Reply:
I guess I don't understand the purpose of this because the package name is already correct.

Reply:
Ok, Maybe we also want to display `Npm Package Name` as well? Since that is the artifact that gets published. So a link on that item that would take you to https://www.npmjs.com/package/@kiali/kiali-ui would be useful to view the dependencies and version info.